frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx

```diff
@@ -48,6 +67,11 @@ export function Card(props) {
             <div className={style["card-content"]}>
               {props.project.content.substring(0, 400)}...
             </div>
+            <div className={style["card-content"]}>
```

Comment:
Can we please make it to appear something like `- <month>, Year` for eg - `January, 2022` and display it at the right end of the card?

Reply:
Yeah, I will try to do it